src/cpp/session/modules/SessionEnvironment.R

```diff
@@ -592,6 +596,10 @@
 {
    obj <- get(objName, env)
    
+   # https://github.com/rstudio/rstudio/issues/9328
+   if (missing(obj))
+      obj <- as.name("Missing argument")
```

Comment:
This is obviously a hack but this felt like the safest change to make here without having to thread more changes throughout this function.